docs/user-guide/street-view.md

```diff
@@ -0,0 +1,23 @@
+# Street View
+*******************
+
+The **Street View** is an extension that can be included in the third step of the application context wizard to allow the end user to browse data from Google street view. The <img src="../img/button/street-view-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provides to the user the pop-up to navigate the map with Google street view. 
```

Comment:
The **Street View** tool allows the user to browse [Google Street View](https://www.google.com/streetview/) contents in MapStore. Through the <img src="../img/button/street-view-button.jpg" class="ms-docbutton"/> button available in the *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, the tool can be activated so that it is possible to navigate the map with [Google Street View](https://www.google.com/streetview/).